scripts/linters/js_ts_linter.py

```diff
@@ -50,8 +50,10 @@
 # NOTE TO DEVELOPERS: Don't add any more files to this list. If you have any
 # questions, please talk to @srijanreddy98.
 INJECTABLES_TO_IGNORE = [
-    'MockIgnoredService', # This file is required for the js-ts-linter-test.
-    'UpgradedServices' # We don't want this service to be present in the index.
+    # This file is required for the js-ts-linter-test.
```

Comment:
This change is done because I introduced a new lint check to prevent trailing/inline comments unless the reason is a valid allowed pragma.